app/views/users/index.html.erb

```diff
@@ -29,17 +29,19 @@
   </div>
 
 </div>
-<div class="row">
-  <div class="col-sm-12">
-    <form class="form-inline users-filter" role="form">
-      <div class="form-group inner-addon left-addon">
-        <label class="sr-only" for="user-filter">Filter users</label>
-        <i class="fa fa-search"></i>
-        <input type="text" class="form-control" id="user-filter" placeholder="Search members...">
-      </div>
-    </form>
-  </div>
-</div>
+<!--the following no longer works with pagination-->
```

Comment:
We could probably get rid of this block of commented code for now. Unless you plan on getting this working again in a near future.